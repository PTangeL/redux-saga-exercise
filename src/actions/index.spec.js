import {
  receiveSession,
  fetchingBeers,
  RECEIVE_SESSION,
  FETCHING_BEERS
} from './'

it(`Returns an Action type ${RECEIVE_SESSION}`, () => {
  const session = {
    "id": "reactJSacademy"
  }
  expect(receiveSession(session)).toMatchSnapshot()
})

it(`Returns an Action type ${FETCHING_BEERS}`, () => {
  const isFetching = {
    isFetching: true
  }
  expect(fetchingBeers(isFetching)).toMatchSnapshot()
})
