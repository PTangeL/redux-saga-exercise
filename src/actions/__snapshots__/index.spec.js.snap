// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns an Action type FETCHING_BEERS 1`] = `
Object {
  "isFetching": Object {
    "isFetching": true,
  },
  "type": "FETCHING_BEERS",
}
`;

exports[`Returns an Action type RECEIVE_SESSION 1`] = `
Object {
  "session": Object {
    "id": "reactJSacademy",
  },
  "type": "RECEIVE_SESSION",
}
`;
